components/FormFlashcard/FormInput.js

```diff
@@ -5,15 +5,24 @@ export default function FormInput({
   maxlength,
   actionMode,
   currentFlashcard,
+  currentCollection,
 }) {
+  console.log("currentFlashcard", currentFlashcard);
+
   return (
     <StyledInput
       id={name}
       name={name}
       maxLength={maxlength}
```

Comment:
Please think about naming camelcase.
```suggestion
      maxLength={maxLength}
```